### PR TITLE
macOS screenshot updates to learning_new_features.rst

### PR DESCRIPTION
**Pull request by CyberSkull**
getting_started/introduction/learning_new_features.rst
* Added macOS screenshot info.
* Keyboard shortcut fixes.
* Better explanation of the code highlighting and clicking.

### CODE REVIEW

File: getting_started/introduction/learning_new_features.rst [hunk 1]
```diff
@@ -121,10 +121,12 @@ information:
    Also, please don't take a picture with your phone, the low quality and screen
    reflections can make it hard to understand the image. Your operating system
    should have a built-in tool to take screenshots with the :kbd:`PrtSc` (Print
-   Screen) key.
+   Screen) key (macOS: use :kbd:`⌘ Command + ⇧ Shift + 3` for a full screen shot).
 
    Alternatively, you can use a program like `ShareX <https://getsharex.com/>`_
-   on Windows or `FlameShot <https://flameshot.org/>`_ on Linux.
+   on Windows, `FlameShot <https://flameshot.org/>`_ on Linux
```
Comment (AThousandShips) [inline getting_started/introduction/learning_new_features.rst:127]:
```suggestion
   on Windows, `FlameShot <https://flameshot.org/>`_ on Linux,
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).

File: getting_started/introduction/learning_new_features.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ explore broad topics while the search bar will help you find more specific
 pages. If a page exists for a given theme, it will often link to more related
 content.
 
-.. image:: img/manual_search.png
+.. image:: img/manual_search.webp
```
Comment (skyace65) [inline getting_started/introduction/learning_new_features.rst:24]: please don't change images to webp just for the sake of making them a webp file, we don't do that.

File: getting_started/introduction/learning_new_features.rst [hunk 3]
```diff
@@ -121,10 +121,12 @@ information:
    Also, please don't take a picture with your phone, the low quality and screen
    reflections can make it hard to understand the image. Your operating system
    should have a built-in tool to take screenshots with the :kbd:`PrtSc` (Print
-   Screen) key.
+   Screen) key (macOS: use :kbd:`⌘ Command + ⇧ Shift + 3` for a full screen shot).
 
    Alternatively, you can use a program like `ShareX <https://getsharex.com/>`_
-   on Windows or `FlameShot <https://flameshot.org/>`_ on Linux.
+   on Windows, `FlameShot <https://flameshot.org/>`_ on Linux,
+   or `Screenshot <https://support.apple.com/guide/mac-help/take-a-screenshot-mh26782/mac>`_
```
Comment (skyace65) [inline getting_started/introduction/learning_new_features.rst:128]: This information isn't necessary. This paragraph is about extra software for taking screenshots, not using built in OS Tools which are mentioned above.

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: getting_started/introduction/learning_new_features.rst (hunk 2 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst:24]: Reverted.

Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst:24]: Reverted.

File: getting_started/introduction/learning_new_features.rst (hunk 3 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst:128]: Moved it to the section above.

Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst]: I've committed a fix, but it isn't showing up here.

File: getting_started/introduction/learning_new_features.rst (hunk 2 above)
Comment (AThousandShips) [reply getting_started/introduction/learning_new_features.rst]: We do encourage updating the format if the file itself is modified, as per our migration to webp

File: getting_started/introduction/learning_new_features.rst (hunk 3 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst]: It has now shown up.

File: getting_started/introduction/learning_new_features.rst [hunk 4]
```diff
@@ -121,10 +121,11 @@ information:
    Also, please don't take a picture with your phone, the low quality and screen
    reflections can make it hard to understand the image. Your operating system
    should have a built-in tool to take screenshots with the :kbd:`PrtSc` (Print
-   Screen) key.
+   Screen) key (macOS: use :kbd:`⌘ Command + ⇧ Shift + 3` for a full screen shot,
```
Comment (skyace65) [inline getting_started/introduction/learning_new_features.rst:124]:
```suggestion
   Screen) key (macOS: use :kbd:`cmd + Shift + 3` for a full screen shot,
```

File: getting_started/introduction/learning_new_features.rst [hunk 5]
```diff
@@ -56,8 +56,8 @@ A class reference's page tells you:
           <https://github.com/godotengine/godot-docs/issues>`_ GitHub repository
           to report it.
 
-You can Ctrl-click (Cmd-click on MacOS) any underlined text like the name of a class, property,
-method, signal, or constant to jump to it.
+You can hold :kbd:`Ctrl` (macOS :kbd:`⌘ Command`) and then mouseover text like the name of a class, property,
+method, signal, or constant to underline it, then :kbd:`Ctrl + Click` (macOS :kbd:`⌘ Command + Click`) it to jump to it.
```
Comment (skyace65) [inline getting_started/introduction/learning_new_features.rst:60]:
```suggestion
You can hold :kbd:`Ctrl` (macOS :kbd:` Cmd`) and then mouseover text like the name of a class, property,
method, signal, or constant to underline it, then :kbd:`Ctrl + Click` (macOS :kbd:`Cmd + Click`) it to jump to it.
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
I Talked to one of the other maintainers, we don't want to do symbols next to keys like this. Also please keep Command shortened to Cmd to be consistent with the rest of the documentation.

File: getting_started/introduction/learning_new_features.rst (hunk 4 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst:124]: Now is it to be `Cmd` or `cmd`? Control is written capitalized as well.

File: getting_started/introduction/learning_new_features.rst (hunk 2 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst]: Also reverted the image as well.

File: getting_started/introduction/learning_new_features.rst (hunk 4 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst]: For now I've done them all as `Cmd` and `Opt`.

File: getting_started/introduction/learning_new_features.rst (hunk 5 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst]: Done.

File: getting_started/introduction/learning_new_features.rst (hunk 2 above)
Comment (CyberSkull) [reply getting_started/introduction/learning_new_features.rst]: I didn't upload this image, so as I understand it, the image should not be converted in this pull request.

**Comment by CyberSkull** [comment]
> I Talked to one of the other maintainers, we don't want to do symbols next to keys like this. Also please keep Command shortened to Cmd to be consistent with the rest of the documentation.

Done.

**Review by skyace65 — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks for contributing! 🎉

**Pull request merged**